libs/ActivityKit/source/ActivityKit.cpp

```diff
@@ -34,6 +34,15 @@ void ActivityKit::stop()
 	_current_activity = nullptr;
 }
 
+void ActivityKit::displayMainMenu(const MagicCard &card)
+{
+	if (card.getLanguage() == Language::fr_FR) {
+		_videokit.displayImage("/fs/home/img/system/robot-misc-choose_activity-fr_FR.jpg");
+	} else {
+		_videokit.displayImage("/fs/home/img/system/robot-misc-choose_activity-en_US.jpg");
+	}
+}
```

Comment:
shouldn't we check first if the card is indeed the dire_roll?

```suggestion
void ActivityKit::displayMainMenu(const MagicCard &card)
{
	if (card != MagicCard::dire_roll) {
		return;
	}
	
	if (card.getLanguage() == Language::fr_FR) {
		_videokit.displayImage("/fs/home/img/system/robot-misc-choose_activity-fr_FR.jpg");
	} else {
		_videokit.displayImage("/fs/home/img/system/robot-misc-choose_activity-en_US.jpg");
	}
}
```